src/App.css

```diff
@@ -1,16 +1,33 @@
-.app {
-  background-color: white;
-} 
-
-.app .editor-overlay {
-  width: 100%;
-  height: 100%;
+.app > * {
+  background: #2D2026;
+  color: #000;
 }
 
 .app .pane-1 {
-  background-color: rgba(211, 209, 209, 0.301);
+  background-color: rgba(211, 209, 209, 0.101);
+  min-width: 350px;
 }
 
 .app .pane-2 {
-  border: solid rgba(0, 0, 0, 0.795) 5px;
+  background-color: #fff;
+  padding: 5em;
+}
+
+.app .editor-overlay {
+  border: 1px solid red;
```

Comment:
Are we still having this border, or was it meant for debugging only?

Reply:
I have no clue 🤣. @bhajneet Do you know?

Reply:
Yes we're keeping the border